src/pages/callforsponsors.mdx

```diff
@@ -21,7 +21,7 @@ Second Annual JavaScript and Friends Conference a success. And with your contrib
 Our team has created opportunities for sponsorship at a number of levels to make it easier for you to support our team of volunteers
 in bringing this about.
 
-Stay tuned for 2020 sponsorship opportunities.
+<a href="https://1drv.ms/b/s!AqhcZadKop7ejCkSer63DidRoPx_" target="_blank" rel="noopener noreferrer">View our Sponsorship Opportunities</a>

```

Comment:
This file should probably either reside in the repository, or made public from the Google Suite files.